swap/htlc.go

```diff
@@ -65,10 +76,15 @@ func (h HtlcOutputType) String() string {
 }
 
 // NewHtlc returns a new instance.
-func NewHtlc(cltvExpiry int32, senderKey, receiverKey [33]byte,
+func NewHtlc(version ScriptVersion, cltvExpiry int32,
```

Comment:
nit: line wrapping looks a bit off (could be mistaken)

Reply:
just checked, it seems correct to me (just added `scriptVersion` and formatted to align with the existing format)